scripts/test.sh

```diff
@@ -6,9 +6,14 @@ if [ -z "$TEST_GREP" ]; then
 fi
 
 node="node"
+runParallel=""
 
 if [ "$TEST_DEBUG" ]; then
    node="node --inspect --debug-brk"
 fi
 
-$node node_modules/.bin/jest "$TEST_GREP" --i
+if [ "$CI" ]; then
+  runParallel="--i"
```

Comment:
Think you're looking at an old commit, https://github.com/babel/babel/pull/5359/files#diff-bbdd5e4087a41b0c0c50414c0362f102R16